slave/process-scim/changelog

```diff
@@ -0,0 +1,5 @@
+perun-slave-process-scim (3.0.0) stable; urgency=low
+
+  * Added service perun-slave-scim
```

Comment:
I know there is already "scim" in the name of service, but i would prefer to have purpose of this slave script in this message too, please.

Reply:
Description added.